WordPress/Classes/ViewRelated/Gutenberg/EditorMediaUtility.swift

```diff
@@ -70,10 +70,16 @@ class EditorMediaUtility {
             size.width = size.width * scale
             requestURL = WPImageURLHelper.imageURLWithSize(size, forImageURL: requestURL)
             request = URLRequest(url: requestURL)
-        } else {
+        } else if allowPhotonAPI {
```

Comment:
@etoledom Can you clarify when allowPhotonAPI is set to true/false is there a related Gutenberg-Mobile PR where I can see the use of this argument?

Reply:
The previous behavior was to always fall back to the Photon API. With this change and setting `allowPhotonAPI = true` as default, all other instances continue working in the same way.

This PR uses `allowPhotonAPI = false` to fetch the image from the original URL directly and avoid the Photon resizing that affects the RN `Image.getSize` method.

Reply:
I'm giving this a second look trying to avoid introducing `allowPhotonAPI`.

Reply:
So, playing with the image scale we are able to get the desired result using the Photon Helper.
I believe it's ready for another 👀 @SergioEstevao 🙏